src/pages/IssueNft/components/IssueNftForm.tsx

```diff
@@ -0,0 +1,209 @@
+import classNames from 'classnames';
+import Select from 'react-select';
+import { Button, MxLink } from 'components';
+import { DataTestIdsEnum } from 'localConstants';
+import { routeNames } from 'routes';
+import { NftEnumType } from 'types';
+import { useIssueNftForm } from '../hooks';
+import { IssueNftFieldsEnum } from '../types';
+
+export const IssueNftForm = () => {
+  const { formik, isLoading, collections, selectedCollection } =
+    useIssueNftForm();
+
+  return (
+    <form
+      onSubmit={formik.handleSubmit}
+      noValidate
+      className='d-flex flex-column'
+    >
+      <div className='flex flex-col gap-4 h-full'>
+        <div className='flex flex-row align-middle w-full'>
+          <div className='flex flex-col w-full'>
+            <Select
+              className='text-sm text-gray-700 placeholder-gray-400 w-full'
+              isLoading={isLoading}
+              options={collections as any}
+              name={IssueNftFieldsEnum.collection}
+              onChange={(option) =>
+                formik.setFieldValue(IssueNftFieldsEnum.collection, option)
+              }
+              onBlur={() =>
+                formik.setFieldTouched(IssueNftFieldsEnum.collection, true)
+              }
+              value={formik.values[IssueNftFieldsEnum.collection]}
+            />
+            {formik.touched[IssueNftFieldsEnum.collection] &&
+              formik.errors[IssueNftFieldsEnum.collection] && (
+                <div
+                  className='text-red-600 text-sm mt-1'
+                  data-testid={DataTestIdsEnum.collectionError}
+                >
+                  {formik.errors[IssueNftFieldsEnum.collection] as any}
+                </div>
+              )}
+          </div>
+          <div className='w-1/3'>
+            <MxLink to={routeNames.issueCollection}>Issue collection</MxLink>
+          </div>
+        </div>
+        <div className='flex flex-col'>
+          <label
+            htmlFor={IssueNftFieldsEnum.name}
+            className='block text-sm font-bold mb-2'
+          >
+            Name:
+          </label>
+          <input
+            className={classNames(
+              'block w-full p-2 text-sm text-gray-700 placeholder-gray-400 border border-gray-300 rounded',
+              {
+                'border-red-600':
+                  formik.touched[IssueNftFieldsEnum.name] &&
+                  formik.errors[IssueNftFieldsEnum.name]
+              }
+            )}
+            data-testid={DataTestIdsEnum.nameInput}
+            id={IssueNftFieldsEnum.name}
+            name={IssueNftFieldsEnum.name}
+            onBlur={formik.handleBlur}
+            onChange={formik.handleChange}
+            placeholder='Enter name'
+            value={formik.values[IssueNftFieldsEnum.name]}
+          />
+          {formik.touched[IssueNftFieldsEnum.name] &&
```

Comment:
a helper like getHasError(formik, IssueNftFieldsEnum.name) would be shorter than
formik.touched[IssueNftFieldsEnum.name] &&
                  formik.errors[IssueNftFieldsEnum.name]

Reply:
Agree on this one. 👍